pages/docs/cloud/repository-integration.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: "Grouparoo Cloud Repository Integration"
+date: "2021-11-10"
+pullQuote: "Simple integration steps for Grouparoo Cloud"
+---
+
+If you are here you should already have an API Key and Project ID.
+
+Grouparoo Cloud requires three things to function: A github action yaml file and two secrets to identify and authorize the code config repository.
+
+### Github Repository Secrets
+
+Grouparoo Cloud requires two secrets to be available in your repository: `GROUPAROO_CLOUD_TOKEN` and `GROUPAROO_CLOUD_PROJECT_ID`.
+
+If these values have already been provided for you, skip ahead. Otherwise you can get them from [Grouparoo Cloud](https://cloud.grouparoo.com/) in your Project's Integrations page.
+
+In your Github repository go to: Settings -> Secrets -> New repository secret.
+
+Add `GROUPAROO_CLOUD_TOKEN` and `GROUPAROO_CLOUD_PROJECT_ID` with the provided values.
```

Comment:
Maybe we can display a table of sorts that has a short description on what each secret is

Reply:
Do we have a particular pattern for this?

Reply:
Not really, but I think now that we have the Integrations page and mentioning that you can get them from there makes more sense. So maybe it's not needed anymore.

Reply:
I could just say `Copy the values from your Project's integrations page`?